tools/src/bin/benchmark-clvm-cost.rs

```diff
@@ -98,23 +101,17 @@ fn quote(a: &mut Allocator, v: NodePtr) -> NodePtr {
     a.new_pair(a.one(), v).unwrap()
 }
 
-fn subst_node(arg: NodePtr, substitution: NodePtr) -> NodePtr {
-    if arg == NodePtr::hack(VARIABLE_VAL) {
-        substitution
-    } else {
-        arg
-    }
+fn subst(arg: Option<NodePtr>, substitution: NodePtr) -> NodePtr {
```

Comment:
Do we need this function?

Reply:
well, it saves a few characters when it's called

Reply:
I would approve a PR that removes it :)